x-pack/legacy/plugins/apm/public/components/app/TransactionDetails/WaterfallWithSummmary/TraceSummary.tsx

```diff
@@ -0,0 +1,255 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { EuiTitle, EuiText, EuiBadge, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { idx } from '@kbn/elastic-idx';
+import moment from 'moment';
+import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
+import styled from 'styled-components';
+import { Transaction } from '../../../../../typings/es_schemas/ui/Transaction';
+import { asTime, asPercent } from '../../../../utils/formatters';
+import { NOT_AVAILABLE_LABEL } from '../../../../../common/i18n';
+import { isRumAgentName } from '../../../../../common/agent_name';
+import { ErrorCountBadge } from './ErrorCountBadge';
+import { truncate } from '../../../../style/variables';
+
+// TODO: Light/Dark theme (@see https://github.com/elastic/kibana/issues/44840)
+const theme = euiLightVars;
+
+const Heading: React.FunctionComponent = () => (
+  <EuiTitle size="xs">
+    <h5>
+      {i18n.translate('xpack.apm.transactionDetails.traceSampleTitle', {
+        defaultMessage: 'Trace sample'
+      })}
+    </h5>
+  </EuiTitle>
+);
+
+const Item = styled('span')`
+  white-space: nowrap;
+`;
+
+interface TimestampProps {
+  transactionTimestamp: string;
+}
+const Timestamp: React.FunctionComponent<TimestampProps> = ({
+  transactionTimestamp
+}) => {
+  const timestamp = moment(transactionTimestamp).format(
+    'MMMM Do YYYY, HH:mm:ss.SSS'
+  );
+
+  return (
+    <Item>
+      <EuiToolTip
+        content={i18n.translate('xpack.apm.transactionDetails.timestampLabel', {
+          defaultMessage: 'Timestamp'
+        })}
+      >
+        <>{timestamp}</>
+      </EuiToolTip>
+    </Item>
+  );
+};
+
+const SeparatorText = styled(EuiText)`
+  color: ${theme.euiColorMediumShade};
+  display: inline;
+`;
+const Separator: React.FunctionComponent = () => (
+  <>
+    {' '}
+    <SeparatorText aria-hidden={true}>|</SeparatorText>{' '}
```

Comment:
Instead of using whitespace `{' '}` for styling, you should add padding to the styles.